packages/web/src/components/admin_page/s3_page/s3.css

```diff
@@ -5,7 +5,8 @@
 
 .s3-upload-image {
     background-color: gray;
-    height: 80vh;
+    padding-top: 4vw;
```

Comment:
Why is this needed?

Reply:
Otherwise the back button will appear on top of the text of the page. With the padding it moves it down a bit so that the button does not overlap with anything